requirements/static-analysis-requirements.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with Python 3.8
 # by the following command:
 #
-#    pip-compile --output-file=requirements/tox-static-analysis-requirements.txt --resolver=backtracking requirements/tox-static-code-analysis.in
+#    pip-compile --output-file=requirements/static-analysis-requirements.txt --resolver=backtracking requirements/static-analysis.in
```

Comment:
There is no need for multiple test environments. It is very difficult to maintain. I removed all and kept just one of them